app/models/notification.rb

```diff
@@ -4,4 +4,22 @@ class Notification < ActiveRecord::Base
   belongs_to :user
 
   validates_presence_of :info_request_event, :user, :frequency, :send_after
+
+  before_validation :calculate_send_after
+
+  INSTANTLY = 'instantly' # Notify me about this as soon as possible
+  DAILY = 'daily' # Notify me about this at most once a day
```

Comment:
I've done this in 28dc63d and I'll just remove 284501d when I squash everything down.

Note to test this, you'll need to run `rake db:rollback STEP=2` in both your test and dev dbs *before* you pull my changes, and then pull and run `rake db:migrate`.